### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by rowanseymour**
* Pin Django to < 1.9
* Update to latest smartmin
* Update to latest rapidpro-python
* Import tidy up

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
 -e git+https://github.com/nyaruka/smartmin.git@master#egg=smartmin-1.8.1
+-e git+https://github.com/rapidpro/rapidpro-python.git@master#egg=rapidpro-python-2.0
 celery-with-redis
-django>=1.7.2
-django-compressor
+django>=1.7.2,<1.9
+django-compressor<2.0
```
Comment (rowanseymour) [inline requirements/base.txt]: 2.x versions of compressor don't work with Django 1.7

**Comment by rowanseymour** [comment]
@norkans7 @nicpottier review please

File: dash_test_runner/tests.py [hunk 2]
```diff
@@ -377,18 +376,24 @@ def test_org_model(self):
...
             client = self.org.get_temba_client()
-            self.assertIsInstance(client, TembaClient)
+            self.assertIsInstance(client, TembaClient1)
             self.assertEqual(client.root_url, 'https://rapidpro.io/api/v1')
             self.assertEqual(client.headers['Authorization'], 'Token %s' % self.org.api_token)
             self.assertEqual(client.headers['User-Agent'], 'test/0.1 rapidpro-python/%s' % client_version)
 
+        client = self.org.get_temba_client(api_version=2)
+        self.assertIsInstance(client, TembaClient2)
+        self.assertEqual(client.root_url, 'http://localhost:8001/api/v1')
```
Comment (norkans7) [inline dash_test_runner/tests.py]: I think this should be v2 link or in this case force to use SITE_API_HOST instead API_ENDPONT

Comment (rowanseymour) [inline dash_test_runner/tests.py]: Good catch. Currently `SITE_API_HOST` is allowed to be a hostname or a complete URL ending with "..api/v1".. which is pretty useful for running in a dev environment. I think we update the client to take URLs without the "..api/v1" ending (e.g. http://localhost:8000/) and have it add "api/v1" or "api/v2" depending on requested version.

Comment (norkans7) [inline dash_test_runner/tests.py]: yep

**Comment by nicpottier** [comment]
yep

**Pull request merged**